controllers/mirrorpeer_controller.go

```diff
@@ -106,6 +108,12 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 	logger.V(2).Info("All validations for MirrorPeer passed", "MirrorPeer", req.NamespacedName)
 
+	if mirrorPeer.Labels == nil {
+		mirrorPeer.Labels = make(map[string]string)
+	}
+	logger.Info("Adding label to mirrorpeer")
+	mirrorPeer.Labels[hubRecoveryLabel] = "resource"
```

Comment:
We never send update request for MirrorPeers. So these labels won't be added.
Need an equivalent of `oc label mirrorpeer` which will append this label (if not present).